Assets/Scripts/Game/EffectManager.cs

```diff
@@ -153,5 +153,41 @@ public void EntityFired(ProjectileEntity projectile)
 				effect.EffectCollection.OnFired(owner as WeaponEntity, projectile, effect.Level);
 			}
 		}
+
```

Comment:
public method, should have a comment here describing what specifically this does